requirements/default.txt

```diff
@@ -2,3 +2,4 @@ celery>=3.1.0
 tornado>=4.0.0
 babel
 pytz
+futures>=3.0.3
```

Comment:
futures is required only for python2. It is added to requirements here https://github.com/mher/flower/blob/master/setup.py#L48